### PR TITLE
Add compatibility with DataModel 6.0

### PR DESCRIPTION
**Pull request by Benestar**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Looks good so far

**Comment by thiemowmde** [comment]
+2 when the releases are done.

**Comment by thiemowmde** [comment]
Rebased and confirmed this does not break compatibility with older DataModel versions. +2 when the missing release is done.

File: tests/unit/Deserializers/LegacyItemDeserializerTest.php [hunk 1]
```diff
@@ -173,7 +174,7 @@ public function testGivenStatementWithLegacyKey_itemHasStatement() {
 	public function testGivenLabels_getLabelsReturnsThem( array $labels ) {
 		$item = $this->itemFromSerialization( array( 'label' => $labels ) );
 
-		$this->assertEquals( $labels, $item->getLabels() );
+		$this->assertEquals( $labels, $item->getLabels()->toTextArray() );
```
Comment (thiemowmde) [inline tests/unit/Deserializers/LegacyItemDeserializerTest.php]: What the heck? This is **not** compatible with older DataModel versions!

**Pull request merged**